src/modules/commander/commander_params.c

```diff
@@ -833,7 +833,7 @@ PARAM_DEFINE_INT32(COM_FLIGHT_UUID, 0);
  *
  * @value 0 Hold
  * @value 1 Mission (if valid)
- * @group Mission
+ * @group Commander
```

Comment:
What was your thought when changing that?

Reply:
- They are among the commander parameters.
- They are used in commander and configure commander behavior.
- I thought the `NAV_` prefix just stayed because these parameters were moved from navigator to commander once but the names stayed backward compatible. This is in principle solvable with the parameter conversion.
- I'm assume some parameters like `COM_OBS_AVOID` that were added after moving parameters from navigator to commander even inherited the possibly wrong group from copy-pasting.

Reply:
Ok fair.